.github/workflows/build-ubuntu.yml

```diff
@@ -111,7 +107,7 @@ jobs:
           cd build
           cmake --build . --config Release --target unittest
       - name: Run Godot unit tests
-        if: matrix.client-godot == 'ON' && matrix.os == 'ubuntu-20.04'
+        if: matrix.client-godot == 'ON'
```

Comment:
with line 29 https://github.com/freeorion/freeorion/pull/3784/files#diff-3dc91c5652448e68371fa6d23e2a83da91a66ad49d85a828dcaf8bcab84ded5fR29 is this ever not `ON`?

Reply:
I left variant with Godot OFF for clang.